### PR TITLE
added ayush7614 info

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains Info About Ayush7614

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @paramsiddharth

**Comment by paramsiddharth** [comment]
Hey @Ayush7614 ! 😊 Greetings of the day. Nice to see you here as a campus expert trainee~!

I just want to make sure if you have completed the modules? If so, we can have a recruitment thread for you and add you to the organization.

Edit: If you haven't done the modules yet, ping me here once you're done and I'll approve the PR for tests! :)

**Comment by Ayush7614** [comment]
> Hey @Ayush7614 ! 😊 Greetings of the day. Nice to see you here as a campus expert trainee~!
> 
> I just want to make sure if you have completed the modules? If so, we can have a recruitment thread for you and add you to the organization.
> 
> Edit: If you haven't done the modules yet, ping me here once you're done and I'll approve the PR for tests! :)

Hey @paramsiddharth Yes I have Completed all the modules and my PR is also Merged

**Comment by paramsiddharth** [comment]
@Ayush7614 Makes sense! 🤔 Sometimes there are bugs in the system due to which the campus expert profile badge takes a while to display. Could you ping me once again once the badge appears on your GitHub profile? Sorry for the inconvenience.

If it takes too long, try mentioning it to Juan.

**Comment by Ayush7614** [comment]
Sure

**Comment by paramsiddharth** [comment]
@Ayush7614 Hey Ayush! 😊 Hope you're doing well. Did you get any updates? I remember when I became a campus expert we faced a similar issue and I had to report it to Juan. Either way, let me know if I can help!

**Comment by Ayush7614** [comment]
> @Ayush7614 Hey Ayush! 😊 Hope you're doing well. Did you get any updates? I remember when I became a campus expert we faced a similar issue and I had to report it to Juan. Either way, let me know if I can help!

Hey, @paramsiddharth Thank you for helping me now all things are set, my onboarding is done you can catch me on GCE's official website (https://githubcampus.expert/Ayush7614/) Can't-Wait to Join the Amazing GCE India Community, Tell me the Next Step.

File: public/users/Ayush7614/info.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Ayush Kumar 
+college: ABES ENGINEERING COLLEGE GHAZIABAD UP INDIA
+location: Ghaziabad, India 
+country: India  
+email: ayushknj3@gmail.com@gmail.com
```
Comment (paramsiddharth) [inline public/users/Ayush7614/info.yml]: I think you accidentally entered an incorrect e-mail address here (I see `@gmail.com` twice. Please fix that. :)

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
> > @Ayush7614 Hey Ayush! 😊 Hope you're doing well. Did you get any updates? I remember when I became a campus expert we faced a similar issue and I had to report it to Juan. Either way, let me know if I can help!
> 
> Hey, @paramsiddharth Thank you for helping me now all things are set, my onboarding is done you can catch me on GCE's official website (https://githubcampus.expert/Ayush7614/) Can't-Wait to Join the Amazing GCE India Community, Tell me the Next Step.

@Ayush7614 Hey Ayush, that's perfect! 😊 I just need you to complete a few more steps and you'll be in!
- [x] Accept the invitation to the organization that I'm sending you in a few minutes.
- [ ] Fix the typo I pointed out in the PR review I mentioned above. After that, you can merge the PR once the tests pass.
- [x] Keep an eye for the thread recruitment thread I create for you once you have accepted the organization invitation.

Welcome to the community, Ayush! 😊❤ So happy to have you here.

File: public/users/Ayush7614/info.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Ayush Kumar 
+college: ABES ENGINEERING COLLEGE GHAZIABAD UP INDIA
```
Comment (paramsiddharth) [inline public/users/Ayush7614/info.yml]: You're using the wrong key here. It should be `university`, not `college`. :)

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
I just noticed another error you made which is why the tests were failing. Please check!

**Comment by Ayush7614** [comment]
> I just noticed another error you made which is why the tests were failing. Please check!

Done

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle

**Review by paramsiddharth — APPROVED** [review]
@Ayush7614 Excellent, Ayush! 😊 Go ahead and merge your PR once the tests pass and then head over to the issue I created for you in the affairs section: [affairs#33](https://github.com/gce-india/affairs/issues/33)

**Comment by Ayush7614** [comment]
> @Ayush7614 Excellent, Ayush! 😊 Go ahead and merge your PR once the tests pass and then head over to the issue I created for you in the affairs section: [affairs#33](https://github.com/gce-india/affairs/issues/33)

Thanks, @paramsiddharth helping me throughout the process

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**